src/org/zaproxy/zap/extension/api/NodeJSAPIGenerator.java

```diff
@@ -31,12 +32,17 @@
 
 public class NodeJSAPIGenerator extends AbstractAPIGenerator {
     
+    /**
+	 * Default output directory in zap-api-nodejs project.
+	 */
+    private static final String DEFAULT_OUTPUT_DIR = "../zap-api-nodejs/src/zapv2/";
```

Comment:
Is the path correct? In zaproxy/zap-api-nodejs#1 the files are being added directly to `src` dir (i.e. not to `src/zapv2`).

Reply:
Yes, I didn't see any reason for a zapv2 directory, that's what versioning is for, unless there is something I've missed?

Reply:
I'm good with that, we should drop the `zapv2` from this path then.

Reply:
Just realised I missed this, thought I'd got it, but obviously hadn't
fe8074c